### ISSUE TITLE
1.7.8.3( classic theme) in the mobile version FO(<=6,1") of the site icons with products on all pages are not adaptive

### ISSUE THREAD

**Issue by antonchik777** [PrestaShop/PrestaShop#27511]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

Hi all,

After upgrading from version PS 1.7.8.2 to version 1.7.8.3( classic theme) in the mobile version FO(<=6,1") of the site icons with products on all pages are not adaptive (icons with products too disproportionately narrowed and the button "quick view" goes beyond the product photo).

Sorry if this is a new feature or the theme has already been created before.

Thanks for your comments.
![bug](https://user-images.githubusercontent.com/35896021/151537334-e70b02dc-7136-44f2-a422-5a7c6717fb95.jpg)

### Expected behavior

For screen smaller than 480px (extra small), the product image should not be deformed, the "quick view" button is hidden and we should show one column per row.

### Steps to reproduce

1. Go to the mobile version FO of the site(<=6,1")
2. See deformed and stretched photos, hover over the product photo to see the unadapted "Quick view" button

### PrestaShop version(s) where the bug happened

1.7.8.3

### PHP version(s) where the bug happened

7.4

**Comment by khouloudbelguith** [comment]
Hi @antonchik777,

I manage to reproduce the issue with PS1783.

In fact with PS1783, we have the quick view is displayed
![image](https://user-images.githubusercontent.com/16067358/151540316-69865cfa-3f59-4113-821c-ce172aaede10.png)
![image](https://user-images.githubusercontent.com/16067358/151540397-eda551c1-7b86-4043-a7ed-f201059a101f.png)

But with PS1782, the quick view is not displayed, we have a strange behavior as you can see in this record
![image](https://user-images.githubusercontent.com/16067358/151540779-4c693997-29d3-459c-9c9d-da295fef190d.png)
https://user-images.githubusercontent.com/16067358/151540671-0269626e-2e22-4e61-88ce-b9bb5aeae54b.mp4

Please be aware that due to the high number of reported issues, some bugs might take a very long time to be resolved. If this issue is important to you and you cannot wait for it to be fixed on the project’s own time, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you. If you manage to get the problem fixed, then please contribute the fix back to the project as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.

Thank you

**Comment by NeOMakinG** [comment]
@prestascott what should we do for the quickview on mobile? Can we just let the zoom icon?

**Comment by mparvazi** [comment]
One product per column in small screens.

**Comment by mparvazi** [comment]
```
{include file='catalog/_partials/products.tpl' listing=$listing productClass="col-xs-12 col-sm-6 col-md-4 col-lg-3"}
```
<img src="https://user-images.githubusercontent.com/85633460/151694099-06c314c8-213c-4fe1-8210-5281ec3374f2.png" width="300">

Product information will be hidden with this class in small screens:
```
<div class="highlighted-informations hidden-sm-down">
```

We can show Quickview in small media, but it should be hidden in xsmall media.
So we need to change it to:
```
<div class="highlighted-informations hidden-xs-down">
```


https://user-images.githubusercontent.com/85633460/151694823-0a729bf0-24e5-47b0-a207-01e93a9224b4.mp4

**Comment by NeOMakinG** [comment]
I have a doubt, did we fix this? I feel like it's been fixed by another issue and this one is duplicated

**Comment by atomiix** [comment]
@khouloudbelguith can you have a look? 🙏

**Comment by Progi1984** [comment]
Ping @PrestaShop/qa-functional :)

**Comment by AureRita** [comment]
Hi @antonchik777, 

I tried to reproduce your bug with an emulation of a Samsung s10 ( it's a mobile with 6,1") and currently the overview is adaptative 
as you can see here : 

https://user-images.githubusercontent.com/100699445/162765229-a45b57ea-522f-4ae2-9516-03f05df09424.mp4

If you keep the same issue, can you tell us because actually I can't reproduce it, 

Thank you for your issue

**Comment by mparvazi** [comment]
I think it happens on xs media screen (< 480px or even lower than).
As I commented before we need to show one column per row on xs media screens.

**Comment by khouloudbelguith** [comment]
Hello,

I tried with 178x branch (PS1786) & develop branch (PS8.0.0)
I still have the same issue when resizing the window
**178x**

https://user-images.githubusercontent.com/16067358/163830511-7e25770d-8fb8-4ac9-89db-a501ef5885ee.mp4

**Develop**

https://user-images.githubusercontent.com/16067358/163830528-ca5bb364-68b7-49a2-8de5-ab4f4bf44181.mp4

I tried with PS1781, the quick view is not displayed

**1781**

https://user-images.githubusercontent.com/16067358/163830728-7d2041e2-7cb1-4cd2-bfd4-d6ca754440d8.mp4


Thanks!

**Comment by MeKeyCool** [comment]
Hi @MatShir, @khouloudbelguith and @mparvazi ,

Sorry but I just want to well understand the expected behavior before starting any development.

For screen smaller than 480px (extra small), the product image should not be deformed, the "quick view" button has do disappear and we should show one column per row.

Do we agree expected result ?

**Comment by mparvazi** [comment]
Hello @MeKeyCool 

On mobile devices we don't have hover state, so need to show quickview in a diffrent way as already implemented in hummingbird: 
https://github.com/PrestaShop/hummingbird/pull/295

Currently mobile devices have pixel density 2 or 3, that means most of them have display lower than 400 pixels for viewport.
For example iPhone 11 resolution width is 828px, its pixel density is 2, so the viewport width is 414px.
Try to check your store for 320px, 360px and 400px.

IMO use one column per row for screens (viewport) lower than 480px.

**Comment by MatShir** [comment]
Let's go for ->

> For screen smaller than 480px (extra small), the product image should not be deformed, the "quick view" button has do disappear and we should show one column per row.


The mobile quick view in the classic theme doesn't look like it was made for mobile, so let's fix the issue first it here. If anyone wants to implement a button for quick view and rework the quick view modal, feel free to submit an issue/request. But most of the work has been fix by @mparvazi one the new theme 🤘
